app/assets/stylesheets/active_admin_sidebar.scss

```diff
@@ -0,0 +1,87 @@
+@import "active_admin_sidebar_pure_icons";
+
+body.active_admin {
+  #active_admin_content.left_sidebar, #active_admin_content.collapsed_sidebar {
+    #sidebar {
+      margin-left: 0;
+      display: block;
+    }
+
+    .paginated_collection_contents {
```

Comment:
Selector should have depth of applicability no greater than 2, but was 3<br>Selector `paginated_collection_contents` should be written in lowercase with hyphens